### ISSUE TITLE
Clarify exactly which code should be saved to `bin/diagnostic.js`

### ISSUE THREAD

**Issue by jrhorn424** [ga-wdi-boston/js-basics-diagnostic#220]

**Comment by gaand** [comment]
Could you expand on this?

**Comment by jrhorn424** [comment]
Yes. Three developers (at least) thought that everything in the Markdown file be pasted into the JS file.

Some thought this meant everything, including text. Some thought this meant all code blocks from the MD file. Some thought it meant only question 5 (I thought this was correct).

**Comment by raq929** [comment]
Having the answers in two files is really confusing. Can we have the whole diagnostic in `bin/diagnostic.js`?
